Improve artifacts documentation (#270)
pypa/hatch: docs/config/build.md

### Artifacts

If you want to include files that are [ignored by your VCS](#vcs), such as those that might be created by [build hooks](#build-hooks), you can use the `artifacts` option. This option is semantically equivalent to `include`.

> Note that artifacts are not affected by the `exclude` option.  Artifacts can
> be excluded by using more explicit paths or using the `!` negate operator.
> When using the `!` operator, the negated pattern must come after the more
> generic pattern.

Reviewer: ```suggestion

Note that artifacts are not affected by the `exclude` option. Artifacts can
be excluded by using more explicit paths or by using the `!` negation operator.
When using the `!` operator, the negated pattern(s) must come after the more
generic ones.
```

If you want to include files that are [ignored by your VCS](#vcs), such as those that might be created by [build hooks](#build-hooks), you can use the `artifacts` option. This option is semantically equivalent to `include`.

Note that artifacts are not affected by the `exclude` option. Artifacts can
be excluded by using more explicit paths or by using the `!` negation operator.
When using the `!` operator, the negated pattern(s) must come after the more
generic ones.

=== ":octicons-file-code-16: pyproject.toml"

    ```toml
    [tool.hatch.build]
    artifacts = [
      "*.so",
      "*.dll",

>       "!/foo/*.so"

Reviewer: ```suggestion
      "!/foo/*.so",
```

    artifacts = [
      "*.so",
      "*.dll",

>       "!/foo/*.so"

Reviewer: ```suggestion
      "!/foo/*.so",
```
